modules/alloy/src/main/ts/ephox/alloy/ui/types/SliderTypes.ts

```diff
@@ -164,6 +164,7 @@ export interface SliderSpec extends CompositeSketchSpec {
 }
 
 export interface SliderApis {
+  setValue: (slider: AlloyComponent, value: SliderValue) => void;
```

Comment:
One thing I forgot sorry, but can you also add a changelog entry for this in the Alloy changelog please?

Reply:
Added that changelog entry, can you take a look?

Reply:
LGTM 👍